packages/arb-token-bridge-ui/src/components/MainContent/ExploreArbitrumContent.ts

```diff
@@ -231,7 +231,7 @@ export const ExploreArbitrumContent = {
 } as const
 
 export type ExploreArbitrumDeFiProjectName =
-  typeof ExploreArbitrumContent.defi[number]['name']
+  (typeof ExploreArbitrumContent.defi)[number]['name']
```

Comment:
After a fresh yarn install these come up as prettier issues

Reply:
I don't think that should be happening? After running `yarn install` some extra dependencies must have been added that override our rules.

eslint `prettier/prettier` is throwing if we have no parenthesis here but at the same time when you run `yarn prettier:format` it removes them.

Reply:
update: this is due to synpress package upgrade, it also updates prettier that includes new rules